src/transformers/image_transforms.py

```diff
@@ -278,6 +289,115 @@ def get_resize_output_image_size(
+        vmin (`float`, *optional*):
+            vmin-valued entries are mapped to start color of cmap. If None, value.min() is used.
+        vmax (`float`, *optional*):
+            vmax-valued entries are mapped to end color of cmap. If None, value.max() is used.
+        vmin_perc (`float`, *optional*, defaults to `1.0`):
+            use the `vmin_perc`-th percentile as `vmin` (outlier rejection).
+        vmax_perc (`float`, *optional*, defaults to `99.0`):
+            use the `vmax_perc`-th percentile as `vmax` (outlier rejection).
+        normalize (`bool`, *optional*, defaults to `False`):
+            Apply normalization between [0,1] for the colored image values.
+        cmap (`str`, *optional*, defaults to `gray_r`):
+            matplotlib colormap to use (requires matplotlib).
+        invalid_val (`int`, *optional*, defaults to `-99`):
+            Specifies value of invalid pixels that should be colored as 'background_color'.
+        invalid_mask (`numpy.ndarray`, *optional*):
+            Boolean mask for invalid regions.
+        background_color (`Tuple[int]`, *optional*, defaults to `(128, 128, 128, 255)`):
+            4-tuple RGB color to give to invalid pixels.
+        gamma_corrected (`bool`, *optional*, defaults to `False`):
+            Apply gamma correction to colored image.
+        value_transform (Callable, *optional*):
+            Apply transform function to valid pixels before coloring.
+        return_numpy (`bool`, *optional*, defaults to `False`):
+            Whether or not to return the resized image as a numpy array. If False a `PIL.Image.Image` object is
+            returned.
+
+    Returns:
+        `np.ndarray` or `PIL.Image.Image`: Colored depth map of shape `(H, W, 4)` and dtype `uint8`.
+    """
+    if is_torch_tensor(value):
+        value = value.detach().cpu().numpy()
+    if is_numpy_array(value):
+        value = value.copy()
+
+    value = value.squeeze()
+    if invalid_mask is None:
+        invalid_mask = value == invalid_val
+    mask = np.logical_not(invalid_mask)
+
+    # normalize
+    vmin = np.percentile(value[mask], vmin_perc) if vmin is None else vmin
+    vmax = np.percentile(value[mask], vmax_perc) if vmax is None else vmax
+    if vmin != vmax:
+        if normalize:
+            value = (value - vmin) / (vmax - vmin)  # 0..1
+        else:
+            value /= vmax  # ..1
+    else:
+        # Avoid 0-division
+        value *= 0.0
+
+    # squeeze last dim if it exists
+    # grey out the invalid values
+
+    value[invalid_mask] = np.nan
+    if value_transform:
+        value = value_transform(value)
+        # value = value / value.max()
+
+    if is_matplotlib_available():
+        cmapper = colormaps.get_cmap(cmap)
+        value = cmapper(value, bytes=True)  # (nxmx4)
+    else:
+        value = (value.clip(0, 1) * 255).astype("uint8")[:, :, None].repeat(4, axis=-1)
+
+    # img = value[:, :, :]
```

Comment:
This was not resolved. You should either apply the reviewers' suggestions or comment why you're not applying them

Reply:
Oh, sorry for that, I was resolving comments for which I made changes on my local branch so that I could better keep up with what's left to do. If it's more common practice, I can just add reactions to those and resolve them only when I push the changes to remote. Is that better?

Reply:
Yes please!